release_notes/45-release-notes.adoc

```diff
@@ -472,4 +473,12 @@ Additionally, this release addresses the following security vulnerabilities:
 * link:https://access.redhat.com/security/cve/cve-2024-24789[CVE-2024-24789] (Improper handling of certain ZIP files in the `archive/zip` package).
 * link:https://access.redhat.com/security/cve/cve-2024-24790[CVE-2024-24790] (Unexpected behavior in the `net/netip` package's Is methods for IPv4-mapped IPv6 addresses).
 
+[id="about-release-456_{context}"]
+== About release 4.5.6
+
+This release of {product-title-short} includes the following updates and bug fixes:
+
+* Updated Golang and go-git versions.
+* Fixed an issue where the HTML code in column values was rendered in PDFs due to insufficient sanitization.
+
```

Comment:
I would suggest to drop `Updated Golang and go-git versions.` line and the following 
```suggestion

Additionally, this release addresses the following security vulnerabilities:
* link:https://access.redhat.com/security/cve/CVE-2025-21613[CVE-2025-21613] (An argument injection vulnerability was found in the `go-git` package).
* link:https://access.redhat.com/security/cve/CVE-2025-21614[CVE-2025-21614] (Vulnerability allows an attacker to perform denial of service attacks by providing specially crafted responses from a Git server in the `go-git` package).
* link:https://access.redhat.com/security/cve/CVE-2024-45338[CVE-2024-45338] (Denial of service vulnerability in the `golang.org/x/net` package).

```

@YuLi517 could you please confirm that following fixes will be in the 4.5.6?

Reply:
I made a new PR:
https://github.com/stackrox/stackrox/pull/13894